code/datums/mind.dm

```diff
@@ -833,8 +851,6 @@
 				qdel(flash)
 				take_uplink()
 				var/fail = 0
-				var/datum/antagonist/traitor/T = has_antag_datum(/datum/antagonist/traitor)
-				fail |= !T.equip_traitor(src)
```

Comment:
Intentional removal?

Reply:
Yeah, it's because at no point do we add a traitor datum to revolutionaries. Which means this would runtime or just not get ran at all.